src/pydata_sphinx_theme/utils.py

```diff
@@ -32,7 +32,9 @@ def config_provided_by_user(app: Sphinx, key: str) -> bool:
     return any(key in ii for ii in [app.config.overrides, app.config._raw_config])
 
 
-def traverse_or_findall(node: Node, condition: str, **kwargs) -> Iterator[Node]:
+def traverse_or_findall(
+    node: Node, condition: Union[Callable, type], **kwargs
```

Comment:
Oops this doesn't seem related to the other changes. Could you move this to a different PR please?